src/main/java/com/o19s/es/ltr/action/TransportListStoresAction.java

```diff
@@ -61,20 +61,14 @@
     private final Client client;
 
     @Inject
-    public TransportListStoresAction(Settings settings, TransportService transportService,ClusterService clusterService,
-                                     ThreadPool threadPool, ActionFilters actionFilters,
-                                     IndexNameExpressionResolver indexNameExpressionResolver, Client client) {
+    public TransportListStoresAction(Settings settings, TransportService transportService,
+                                     ClusterService clusterService, ThreadPool threadPool, ActionFilters actionFilters,
+                                     IndexNameExpressionResolver indexNameExpressionResolver, String executor, Client client) {
         super(ListStoresAction.NAME, transportService, clusterService, threadPool,
-            actionFilters, ListStoresActionRequest::new, indexNameExpressionResolver);
+            actionFilters, ListStoresActionRequest::new, indexNameExpressionResolver, ListStoresActionResponse::new, ThreadPool.Names.SAME);
```

Comment:
We may want to use the new `executor` param.

Reply:
Whoops didn't mean to resolve this, I has the swapped in for ThreadPool at some point, but I will put `executer` back in there tonight

Reply:
I tried passing it thru and it caused tests to fail out so I'm sticking with the SAME for now.